www/index.html

```diff
@@ -1,46 +1,172 @@
+
+    <style is="custom-style">
+        paper-toolbar.custom {
+            --paper-toolbar-background: green;
+        }
+
+        paper-spinner.multi {
+            position: absolute;
+            margin-left: 75%;
+            margin-top: 5%;
+            z-index: 1000;
+        }
+    </style>
+
+</head>
+
+<body>
+
+<script src="scripts/app.js"></script>
+<script type="text/javascript" charset="utf-8">
+
+    document.addEventListener('WebComponentsReady', function () {
+        var button = document.querySelector('paper-button');
+
+        button.addEventListener('click', function () {
+            var spinner = document.querySelector('paper-spinner');
+            spinner.active = true;
+
+            var options = {
+                enableHighAccuracy: true,
+                maximumAge: 0
+            };
+
+            navigator.geolocation.getCurrentPosition(geolocationSuccess, geolocationError, options);
+        });
+
+        function geolocationError(error) {
+            console.warn('ERROR(' + error.code + '): ' + error.message);
+
+            var toast = document.querySelector('paper-toast');
+            toast.text = 'something went wrong with your geolocation, please try to activate it or authorize the app and retry :)';
+            toast.duration = 5000;
+            toast.open();
+        }
+
+        function geolocationSuccess(position) {
+
+            var spinner = document.querySelector('paper-spinner');
+            spinner.active = false;
+
+            var crd = position.coords;
+
+            var gMap = document.querySelector('google-map');
+            gMap.fitToMarkers = true;
+
+            var start_lat = crd.latitude;
+            var start_long = crd.longitude;
+
+            var start = start_lat + ',' + start_long;
+            var end = '48.2919872,4.0805841';
+
+            document.querySelector('google-map-directions').map = gMap.map;
+            document.querySelector('google-map-directions').startAddress = start;
+            document.querySelector('google-map-directions').endAddress = end;
+
+        }
+    });
+
+</script>
+
+
+<paper-drawer-panel>
+    <div drawer>
+        <paper-toolbar class="custom">
+            <div>What bar are you looking for ?</div>
+        </paper-toolbar>
+
+
+    </div>
+
+    <div main>
+        <paper-spinner class="multi"></paper-spinner>
+
+        <paper-toolbar class="custom">
+            <paper-icon-button icon="menu" paper-drawer-toggle></paper-icon-button>
+            <div class="title">Happly Hour</div>
+        </paper-toolbar>
+
+        <paper-button id="button_find_my_bar" raised>For now, Find The Berthom <3 !</paper-button>
+
+        <google-map latitude="37.779" longitude="-122.3892" api-key="AIzaSyAMK9GqpUJOOGk6O2mjxKKQ7TTlfIJx_yQ"></google-map>
+        <google-map-directions start-address="" end-address="" travel-mode="WALKING"
+                               api-key="AIzaSyAMK9GqpUJOOGk6O2mjxKKQ7TTlfIJx_yQ"></google-map-directions>
```

Comment:
Actually it's ok, Google provides other ways to secure the usage of their api key :)
You can keep that.